src/openads/infrastructure/connector/appnexus/AppNexusConnectorImpl.js

```diff
@@ -48,17 +51,17 @@ export default class AppNexusConnectorImpl extends AppNexusConnector {
 
   loadTags () {
     this._logger.debug('loadTags has been requested')
-    if (this._debounce !== undefined) clearTimeout(this._debounce)
+    if (this._debounceTimeOut !== undefined) clearTimeout(this._debounceTimeOut)
     this._loadTagsDebounceOperator()
     return this
   }
 
   _loadTagsDebounceOperator () {
-    this._debounce = setTimeout(() => {
+    this._debounceTimeOut = setTimeout(() => {
```

Comment:
Do not create properties on the fly :(
Also the property name should be something like TimerId or debounceTimerId ;)